python/modules/Stop0lObjectsProducer.py

```diff
@@ -229,8 +231,7 @@ def analyze(self, event):
         self.IsoTrack_Stop0l = map(lambda x : self.SelIsotrack(x, met), isotracks)
         self.Tau_MtW	     = map(lambda x : self.CalMtW(x, met), taus)
         self.Jet_Stop0l      = map(self.SelJets, jets)
-        local_BJet_Stop0l    = map(self.SelBtagJets, jets)
-        self.BJet_Stop0l     = [a and b for a, b in zip(self.Jet_Stop0l, local_BJet_Stop0l )]
```

Comment:
What is this change about?

Reply:
no using the Jet_Stop0l (30GeV cut now). So that Btag is selected with 20GeV cut